packages/test-runner-saucelabs/src/createSauceLabsLauncher.ts

```diff
@@ -33,39 +35,56 @@ export function createSauceLabsLauncher(
   }
   const manager = new SauceLabsLauncherManager(finalSauceLabsOptions, finalConnectOptions);
 
-  return function sauceLabsLauncher(capabilities: Record<string, unknown>): BrowserLauncher {
+  return function sauceLabsLauncher(capabilities: DesiredCapabilities): BrowserLauncher {
     if (capabilities == null) {
       throw new Error('Capabilities are required.');
     }
-    const finalCapabilities = { ...capabilities };
 
-    // sync the tunnel identifier, username and access key
-    finalCapabilities['sauce:options'] = {
-      ...(finalCapabilities['sauce:options'] as Record<string, unknown>),
-      username: saucelabsOptions.user,
-      accessKey: saucelabsOptions.key,
+    let finalCapabilities = { ...capabilities };
+
+    const finalSauceCapabilities = {
       tunnelIdentifier: finalConnectOptions.tunnelIdentifier,
+      ...saucelabsCapabilities,
+    };
+
+    // W3C capabilities: only browserVersion is mandatory, platformName is optional.
+    // Note that setting 'sauce:options' forces Sauce Labs to use W3C capabilities.
+    if (capabilities.browserVersion) {
+      finalCapabilities.platformName =
```

Comment:
Is this needed to fix a bug? Otherwise I prefer that the defaults are set by saucelabs, not by us.

Reply:
If you set `browserVersion` with `platform`(instead of `platformName`), then the following error will happen:

```
"message": "Illegal key values seen in w3c capabilities: [platform]",
"error": "unknown error",
```

Removed the fallback to `Windows 10` to let Sauce Labs choose the default platform as you suggested.